BibBuddy/app/src/main/java/de/bibbuddy/NoteDAO.java

```diff
@@ -139,19 +140,18 @@ public void delete(Long id) {
 	 * @param type       note type (text, voice, image)
 	 * @param text       note text if it is of type text
 	 * @param createDate note creation date
-	 * @param modDate    note modification date
 	 * @param noteFileId note file id
 	 */
-	public void updateNote(Long id, String name, int type, String text, Long createDate, Long modDate,
-												 Long noteFileId) {
+	public void updateNote(Long id, String name, int type, String text, Long createDate, Long noteFileId) {
+		Long currentTime = new Date().getTime();
```

Comment:
If Note is update in the DB the mod_date is automatically updated and the create_date must only be set if the Note is created at first time.

Reply:
There is no any other modDate update for me - ModDate is only assigned when a new note is created manually in NoteDAO. After that it needs to be changed to the value that matches the moment the user stops editting it (leaving TextNoteEditorFragment) -for this purpose there is only a NoteDAO.update() used, which is using database.update()-method, which needs all contentValues as params, that need to be changed (including the modDate). 
Why the createDate is assigned to the change-contentValues: I only assumed it's better to put all values for each column of the row into the contentValues that are put into the database, I think we can remove that though (but we should do that for all never changing values then (this means: Id, Type, CreateDate and NoteFileId).